### ISSUE TITLE
Single responsibility

### ISSUE THREAD

**Issue by dmarcos-au** [archit737/code-challenge#4]
https://github.com/archit737/code-challenge/blob/3eaa897237c27d1ccd6aad272f20cbbf0a70be89/src/main/java/com/transurban/utils/cloud/GoogleDriveInteract.java#L58

This is mixing two operations here - one to retrieve the files and the other do display. Ideally, this method would just be responsible for retrieving the files and returning then in a vendor-neutral way. The caller should then be responsible for how to "display" the items. E.g. if we wanted to create a UI for this, ideally we shouldn't need to modify this class.

**Comment by archit737** [comment]
You're absolutely right. Retrieval and rendering logic should be seperate but as we don't have a UI so I decided to combine it with the retrieval. In an ideal scenario (with the UI), the System.out.Println() statement would move out from the method and the method would simply return the data ,say, in a JSON format to a renderer.
